models/members.js

```diff
@@ -98,8 +98,37 @@ const deleteIsMemberProperty = async () => {
   }
 }
 
+/**
+ * Fetches the data about our members with role member:true
+ * @return {Promise<userModel|Array>}
+ */
+
+const fetchOnlyMembers = async () => {
```

Comment:
How is this function different from the `fetchMembers` function defined already?
Can we reuse it?

Reply:
@ankurnarkhede , with the function `fetchMembers` we would be receiving the details of all the users along with their roles defined.
Upon scaling, we would rather be doing filter operation for millions of data which could be handled with db call while fetching required details right thus reducing computational time.
Open to suggestion

Reply:
We do have `fetchUsers` in users model due to which I went with different way of naming convention...
Shall I proceed with changing naming convention of `fetchMembers` to `fetchMembersWithAllRoles` and `fetchOnlyMembers` to `fetchMembersWithMemberRole` ?

Reply:
just go with `fetchUsers`